app/views/bright_events/_filters.html.erb

```diff
@@ -24,7 +24,7 @@
 
   <script>
     function updateMinDate() {
-      var dateFromValue = document.getElementById('date_from').value;
+      let dateFromValue = document.getElementById('date_from').value;
       document.getElementById('date_to').min = dateFromValue;
     }
   </script>
```

Comment:
Move this code to a JavaScript controller using Stimulus. It's not a good practice to include logic in views; separating Ruby or JavaScript logic from views improves maintainability.

Reply:
Done!